src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/GeocodeVacancyCommandHandler.cs

```diff
@@ -50,7 +50,7 @@ public async Task Handle(GeocodeVacancyCommand message, CancellationToken cancel
 
         private Task<Geocode> GeocodePostcodeAsync(Vacancy vacancy)
         {
-            _logger.LogInformation("Attempting to geocode postcode:'{postcode}' for vacancyId:'{vacancyId}'", vacancy.EmployerLocation.Postcode, vacancy.Id);
+            _logger.LogInformation("Attempting to geocode for vacancyId:'{vacancyId}'", vacancy.Id);
```

Comment:
While we are printing postcode here... but it has no association with any individual... so postcode in itself shouldn't be an issue?